flexmeasures/cli/monitor.py

```diff
@@ -34,18 +37,26 @@ def send_monitoring_alert(
             f"Last run was at {latest_run.datetime}, status was: {latest_run.status}"
         )
 
+    custom_msg_txt = ""
+    if custom_msg:
+        custom_msg_txt = f"\n\nNote: {custom_msg}"
+
     capture_message_for_sentry(msg)
 
     email_recipients = app.config.get("FLEXMEASURES_MONITORING_MAIL_RECIPIENTS", [])
     if len(email_recipients) > 0:
         email = Message(subject=f"Problem with task {task_name}", bcc=email_recipients)
-        email.body = f"{msg}\n\n{latest_run_txt}\nWe suggest to check the logs."
+        email.body = (
+            f"{msg}\n\n{latest_run_txt}\nWe suggest to check the logs. {custom_msg_txt}"
+        )
         app.mail.send(email)
 
     app.logger.error(f"{msg} {latest_run_txt}")
+    if custom_msg:
+        app.logger.info(custom_msg)
```

Comment:
Wouldn't it be better if this was part of the error log in line 54? Otherwise, the relevant information for debugging may wind up being dispersed over two separate log files.

Reply:
I thought it might get long, but you're probably right.